scripts/cleanup-braces.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+import re
+import sys, os
+
+DEBUG = True
```

Comment:
Should we still include the `debug` function since nothing calls it? Does it make more sense to disable debug output by default?
```suggestion
DEBUG = False
```